docs/policies.md

```diff
@@ -71,6 +71,33 @@ Members and contractors should consider the following rules when considering tak
 - To be eligible for the monthly healthcare stipend, members must have worked a rolling 1000 hours over the past 12 months (pro-rated for those that have been with the co-op for less than a year).
 - Members and contractors are encouraged and expected to keep their team members informed and update their calendar regularly to reflect their availability.
 
+### Extended Unpaid Health Leave Policy
+
+A member may take up to 12 weeks of unpaid health related leave for any of the following reasons:
+
+- For the birth and care of the newborn child of a member;
+- For placement with the member of a child for adoption or foster care;
+- To care for an immediate family member (i.e., spouse, child, or parent) with a serious health condition; or
+- To take medical leave when the member is unable to work because of a serious health condition.
+
+The member seeking unpaid health leave must notify the Board (<board@catalyst.coop>) as soon as possible. The notification certification should include:
```

Comment:
what is a notification certification?

Reply:
It's what the FMLA calls a doctors note.